compiler/damlc/daml-ide-core/data/prod/hlint.yaml

```diff
@@ -0,0 +1,975 @@
+# hlint configuration file
```

Comment:
You are just trying to increase your commit stats by commiting more and more hlint configs, are you? :wink:

Reply:
No, no, that's not it :)